osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -304,9 +304,9 @@ private void initializeChannels()
 
                 Scheduler.Add(delegate
                 {
-                    addChannel(channels.Find(c => c.Name == @"#lazer"));
-                    addChannel(channels.Find(c => c.Name == @"#osu"));
-                    addChannel(channels.Find(c => c.Name == @"#lobby"));
```

Comment:
?

Reply:
Yeah not sure about this one. Now that joined channels are stored server-side we may not want to force this action any more.

Reply:
Then please remove this lines completly.